assets/app/views/project.html

```diff
@@ -55,19 +62,100 @@ <h2 class="service">{{serviceId}}
+          <div class="row">
+            <div class="col-md-12">
+              <div class="tile">
+                <h2 class="service">
+                  <em>There is no service for these pods.</em>
+                </h2>      
+                <div class="separator"></div>
+                <!-- As long as pods created from the deployments created from this deployment config will not be routed
+                     to by any service, then we will show things related to triggers that are about to create stuff, like builds.
+                     If things created from the deployment config in its current state would be routed to be a service
+                     then any trigger details would be showing up under the service instead.
+                -->
+                <div ng-if="deploymentConfigs[deploymentConfigId] && unservicedDeploymentConfigs[deploymentConfigId]">
+                  <triggers ng-init="triggers = deploymentConfigs[deploymentConfigId].triggers"></triggers>
+                </div>                
+                <div class="ng-scope" style="margin-top: 10px; text-align: center;">
+                  <div class="small muted" style="margin-bottom: 10px;">
+                    <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp>
+                    <span ng-if="deployment.details && deployment.details.causes && deployment.details.causes.length > 0">
+                      <span>, triggered by 
+                        <span ng-repeat="cause in deployment.details.causes">
+                          <span ng-if="cause.type == 'ImageChange'">new image for {{cause.imageTrigger.repositoryName | imageName}}:{{cause.imageTrigger.tag}}</span>
+                          <span ng-if="cause.type == 'ConfigChange'">deployment configuration change</span>
+                        </span>
+                      </span>
+                    </span>                    
+                  </div>                
+                  <div style="display: inline-block;">
+                    <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+                    <pods pods="deploymentPods[deployment.metadata.name]"></pods>
+                  </div>
+                </div>        
+              </div>
+            </div>
+          </div>      
+        </div>
+      </section>
+      <!-- Don't show me succeeded and terminated pods for now, we may want to show some of them but we would definitely want
+           to hide our build pods from these view.  Or we need to actually reap old build pods.
+           Also hide deployer pods.
+       -->
+      <section ng-repeat="(podName, pod) in solitaryPods" ng-if="pod.status.phase != 'Succeeded' && pod.status.phase != 'Terminated' && !buildPods[podName] && !(pod | annotation : 'deployment')">
```

Comment:
Move this logic into the controller method? If you want to keep track of all solitaryPods, as well as the filtered ones we're going to display, you could have `unfilteredSolitaryPods` and `solitaryPods` in the controller, then have the view just render `solitaryPods`.